quality/errorprone/android.gradle

```diff
@@ -1,16 +1,17 @@
 apply plugin: 'net.ltgt.errorprone'
 
 dependencies {
-  if(!JavaVersion.current().java9Compatible) {
-    // see https://github.com/tbroyer/gradle-errorprone-plugin#jdk-8-support
-    errorproneJavac 'com.google.errorprone:javac:9+181-r4173-1'
-  }
-  errorprone 'com.google.errorprone:error_prone_core:2.3.2'
-  annotationProcessor 'com.uber.nullaway:nullaway:0.6.5'
-  testAnnotationProcessor 'com.uber.nullaway:nullaway:0.6.5'
+    if (!JavaVersion.current().java9Compatible) {
+        // see https://github.com/tbroyer/gradle-errorprone-plugin#jdk-8-support
+        errorproneJavac 'com.google.errorprone:javac:9+181-r4173-1'
+    }
+    errorprone 'com.google.errorprone:error_prone_core:2.3.2'
+    annotationProcessor 'com.uber.nullaway:nullaway:0.6.5'
+    testAnnotationProcessor 'com.uber.nullaway:nullaway:0.6.5'
 }
 
 tasks.withType(JavaCompile) {
-  options.errorprone.errorproneArgs << '-XepDisableWarningsInGeneratedCode'
-  options.errorprone.errorproneArgs << '-Xep:NullAway:ERROR' << '-XepOpt:NullAway:AnnotatedPackages=com.uber'
+    // Adding (-Xep:ParameterName:OFF) due to issue: https://github.com/google/error-prone/issues/780.
+    options.errorprone.errorproneArgs << '-XepDisableWarningsInGeneratedCode' << '-Xep:ParameterName:OFF'
+    options.errorprone.errorproneArgs << '-Xep:NullAway:ERROR' << '-XepOpt:NullAway:AnnotatedPackages=com.uber'
```

Comment:
why all the indentation changes? can you retain the original indentation in order to not pollute the diff & history?

- [ ] fix indentation

Reply:
fixed